core/investigation.py

```diff
@@ -87,10 +87,6 @@ def get_form(klass):
     def info(self):
         result = self.to_mongo()
         result['nodes'] = [node.to_mongo() for node in self.nodes]
-        try:
-            result["shared"] = [Group(id=shared_with_id).reload() for shared_with_id in Investigation.objects.get(id=self.id).sharing]
```

Comment:
This seemed to never work since it would try to query for a Group where `shared_with_id` was a user. Please le me know if it makes sense to remove it (beside, the same data was contained in the `sharing` key)